### PR TITLE
Page and Social Override documentation part deux

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?

This PR is a redo of #997 _after_ the GitBooks v2 migration. Re-adding the content added in #997 based on the new GitBook v2 format.

You can view the updated docs for this branch at: <https://dosomething.gitbook.io/phoenix-documentation/v/docs-page-and-social-override-information/>

closes #997 

### What are the relevant tickets/cards?

Refs [Pivotal ID #158394953](https://www.pivotaltracker.com/story/show/158394953)
Refs [Pivotal ID #158394965](https://www.pivotaltracker.com/story/show/158394965)

### CODE REVIEW

File: docs/content-publishing/pages/README.md [hunk 1]
```diff
@@ -5,3 +5,51 @@ You can create pages, like the "FAQ" or "Scholarship" pages, and attach them to
 * [Page Creation](page-creation.md)
 * [Making pages modals](making-pages-modals.md)
 
+# Pages
+
+The 'Page' Content Type is multi-purpose, and has gone through a few phases. We'll list the relavant ways to use a Page:
+
+## 1. Campaign Sub Page
```
Comment (mendelB) [inline docs/content-publishing/pages/README.md]: Do we want to get rid of this part and update the Campaign Page section based on #1007 or save for a follow up PR?

Comment (weerd) [reply docs/content-publishing/pages/README.md]: I'm updating as I go, and switching it over to not refer to anything as "sub page" :)

**Comment by weerd** [comment]
Oh @mendelB didn't realize you 👍'd the PR. I'm still working on it. Should have edited the PR title to indicate it's a WIP. Just wanted to push it up so it'd show up as a viewable branch in GitBooks :)

**Comment by mendelB** [comment]
Oh lol I figured you were maybe taking it piecemeal

**Pull request merged**